### DISCUSSION TITLE
Backend agnostic framework

### DISCUSSION THREAD

**Discussion by Wulfheart** [spatie/ignition#21] [Q&A]
I love the new design and I was wondering if it could be an option to make the react part framework agnostic so it could be also achieved with another language supplying some kind of standardized json and then it gets rendered. The business-logic behind it, that is e.g. the collection of the stack trace, would still have to be implemented by the authors but this package (spatie/ignition) could serve as reference implementation.
What do you think about it?

 cc @reindert-vetter => maybe for [confetti](https://github.com/confetti-framework/confetti)?

**Comment by reindert-vetter** [comment]
Sure, that would be great! Once the API is documented, I think I'll implement it.
